extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -639,8 +639,8 @@ def validate(self, classifier_data):
                     float_property)
             if not isinstance(classifier_data[float_property], float):
                 raise utils.ValidationError(
-                    ('Expected %s to be a float, received %s' %
-                     float_property, classifier_data[float_property]))
+                    ('Expected %s to be a float, received %s' %(
```

Comment:
No, It did not fail lint checks. Since I am using two placeholders, I have to provide two parameters in parenthesis. Without parenthesis, only the first parameter was being considered and the second one was ignored. So, I updated the line with parenthesis.
Upadated with additional space.